Sources/LeafKit/LeafError.swift

```diff
@@ -31,6 +31,8 @@ public struct LeafError: Error {
         /// Attempt to render an AST with cyclical external references
         /// - Provide template name & ordered array of template names that causes the cycle path
         case cyclicalReference(String, [String])
+        /// Parameter missing
+        case missingParameter
```

Comment:
Ok I'm going to come in and ruin the party, but only a little - we can't add a new enum case without it being a breaking change unfortunately. And the LeafError is not really one of the enums that we consider 'private'. The quick fix to this will be to throw one of the existing errors, probably `unknownError`. That will allow us to release it as a patch version

Reply:
And we had so much fun here. Oh well have it your way then ;-)
Updated as per request.